src/test/scala/mesosphere/marathon/core/task/TaskStatusComparisonTest.scala

```diff
@@ -12,29 +12,29 @@ class TaskStatusComparisonTest extends UnitTest with TableDrivenPropertyChecks {
   "A task" when {
 
     // format: OFF
-    // Conditions and their expected flag values. e.g. if status is Condition.Reserved isReserved is expected to be true
+    // Conditions and their expected flag values. e.g. if status is Condition.Created isCreated is expected to be true
     // and isCreated to be false.
     val conditions = Table (
-      ("condition",                   "isReserved", "isCreated", "isError", "isFailed", "isFinished", "isKilled", "isKilling", "isRunning", "isStaging", "isStarting", "isUnreachable", "isUnreachableInactive", "isGone", "isUnknown", "isDropped", "isActive", "isTerminal"),
```

Comment:
are we ready to remove this isReserved too?

Reply:
I think this is not used anywhere. @alenkacz ?

Reply:
isReserved was used only in one place for Task class and that one was removed (because Reserved is now just terminal as any other terminal)